test/dx_mgn_pool.js

```diff
@@ -253,6 +290,10 @@ contract("DxMgnPool", (accounts) => {
       const frtToken = dx.contract.methods.frtToken().encodeABI()
       await dxMock.givenMethodReturnAddress(frtToken, mgnTokenMock.address)
 
+      const owlTokenMock = await MockContract.new()
+      const owlToken = dx.contract.methods.owlToken().encodeABI()
+      await dxMock.givenMethodReturnAddress(owlToken, owlTokenMock.address)
+
```

Comment:
I noticed that these three lines are repeating several times over and over. Do they really need to be in every unit test?

Reply:
Yeah, this is a little bit annoying, but our approach was:
We should be able to read tests as easily as a textbook

It has the beauty that for every test every condition or setting is listed linearly and easy to digest.